testing/binding.yml

```diff
@@ -0,0 +1,25 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
No need to create a separate cluster-role if going to give it access to everything, you can just use `cluster-admin`